frontend/docker/dev/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY ./frontend/yarn.lock ./
 
 # Install packages, use --ignore-scripts to not call postinstall, as it causes this step to fail because config.json
 # is not copied yet, and copying it here will trigger new install if config changes, which is inconvenient for development.
-RUN yarn install --ignore-scripts
+RUN yarn install --ignore-scripts --verbose --network-timeout 100000
```

Comment:
Was this change intentional ?

Reply:
Nope, I am removing it in this [commit](https://github.com/awslabs/aws-dataall/commit/ceefd340f3cf438eac1a6580ced1934611a3ade3), plus I added some logs for the graphql calls. That is intentional, a customer requested those